documentation/modules/ostack-token-auth.adoc

```diff
@@ -0,0 +1,93 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc
+
+:_content-type: PROCEDURE
+[id="openstack-token-authentication_{context}"]
+= Using token authentication with an {osp} source provider
+
+You can use token authentication instead of username/password authentication when you use {project-first} to migrate a virtual machine that has an {osp} source provider.
+
+{project-short} supports both of the following types of token authentication:
+
+* Token with user ID
+* Token with user name
+
+For each type of token authentication, you need to use data from OpenStack to create a `Secret` manifest.
+
+.Prerequisites
+
+Have an {osp} account.
+
+.Procedure
+
+. In the dashboard of the {osp} web console, click *Project > API Access*.
+. Expand *Download OpenStack RC file* and click *OpenStack RC file*.
++
+The file that is downloaded, referred to here as `<openstack_rc_file>` includes the following fields used for token authentication:
++
+[source, terminal]
+----
+OS_AUTH_URL
+OS_PROJECT_ID
+OS_PROJECT_NAME
+OS_DOMAIN_NAME
+OS_USERNAME
+----
+
+. To get the data needed for token authentication, run the following command:
++
+[source,terminal]
+----
+$ openstack token issue
+----
++
+The output, referred to here as `<openstack_token_output>`, includes the `token` `userID`, and `projectID` you need for authentication using a token with user ID.
```

Comment:
@RichardHoch 
Comma after `token`
Also, I think we should not use words in backticks as common nouns, so 'the' should not be here. But up to you.

Reply:
You're right about the comma, but since the YAML file uses token, etc. as names of labels, I think using backticks here is appropriate, but I see your point.

Reply:
@RichardHoch 
I didn't mean comma instead of a backtick, but after it, like the one after `userID`.